aggregate_test.go

```diff
@@ -252,10 +252,11 @@ func Test_Aggregate(t *testing.T) {
 			ErrorMatcher: nil,
 		},
 
-		// Test 4 ensures depending bundles within the given capabilities are not
-		// bundled together within the aggregated state in case their dependency
-		// definitions do not meet their constraints. Thus the aggregated result
-		// should be empty because there is no proper bundle available.
+		// Test 4 ensures depending version bundles within the given list of version
```

Comment:
Can you split this into two sentences? It runs on super long and is hard to understand currently.

Reply:
Yeah that was really weird. I tried to simplify it.

Reply:
Thanks, reads much better now!